src/Api/Auth/Controllers/AccountsController.cs

```diff
@@ -654,6 +671,13 @@ public async Task SetUserVerifyDevicesAsync([FromBody] SetVerifyDevicesRequestMo
         await _userService.SaveUserAsync(user);
     }
 
+    [HttpPost("verify-devices")]
+    [Obsolete("This endpoint is deprecated. Use PUT /verify-devices instead.")]
+    public async Task PostSetUserVerifyDevicesAsync([FromBody] SetVerifyDevicesRequestModel request)
+    {
+        await SetUserVerifyDevicesAsync(request);
+    }
+
```

Comment:
I checked through the other changes and they should all be good, but we use POST on the clients for this method so I've created https://bitwarden.atlassian.net/browse/PM-25376 to have us swap over to the PUT.